cmd/cli/version.go

```diff
@@ -17,9 +17,12 @@ import (
 	"github.com/openservicemesh/osm/pkg/version"
 )
 
-const versionHelp = `
+const (
+	versionHelp = `
 This command prints the OSM CLI and remote mesh version information
 `
+	dev = `dev`
```

Comment:
I'd prefer `devVersion` as the var name to remove ambiguity.

Reply:
Make sense. It is clearer on this. I will change it. Thanks!

Reply:
Changes made and tested in local + AKS. Thanks